batch/batches/MoveCategoryEntries/KAsyncMoveCategoryEntries.class.php

```diff
@@ -187,79 +139,75 @@ private function moveEntries(KalturaBatchJob $job, KalturaMoveCategoryEntriesJob
 	{
 		$categoryEntryFilter = new KalturaCategoryEntryFilter();
 		$categoryEntryFilter->orderBy = KalturaCategoryEntryOrderBy::CREATED_AT_ASC;
-		$categoryEntryFilter->categoryIdEqual = $srcCategoryId;
-		
+		if($data->moveFromChildren) {
```

Comment:
Is there any case that moveFromChildren won't be true?
I think you can remove $srcCategoryId from this method, from KalturaMoveCategoryEntriesJobData and from kMoveCategoryEntriesJobData.